setup.py

```diff
@@ -71,7 +71,7 @@
         'gitpython',
         'jinja2',
         'knack',
-        'pylint==2.11.1',
+        'pylint>=2.12,<=2.17.1',
```

Comment:
```suggestion
        'pylint>=2.12,<2.18',
```
Use this to make sure we are using latest 2.17 version. (2.17.4 has been released)